### PR TITLE
Bug #1747: Display volume in sample detail when the value is 0.

### PR DESCRIPTION
**Pull request by nafiz1001**
https://206.12.92.46/issues/1747

### CODE REVIEW

**Comment by nafiz1001** [comment]
`parseFloat(undefined).toFixed(3)` and `parseFloat(null).toFixed(3)` show up as `Nan` which is not really a big deal given that it's temporary and might even go away very quick.

File: frontend/src/components/samples/details/SampleDetailsProcessMeasurements.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ const SampleDetailsProcessMeasurements = ({processMeasurements, usersByID, proto
         align: 'right',
         render: (volumeUsed, _) =>
             <div>
-              {volumeUsed ? `${parseFloat(volumeUsed).toFixed(3)}` : undefined}
+              {parseFloat(volumeUsed).toFixed(3)}
```
[the remark this answers is not in the document]
Comment (nafiz1001) [reply frontend/src/components/samples/details/SampleDetailsProcessMeasurements.js:51]: should it also test against null?

**Comment by ckostiw** [comment]
There are some fishy parseFloats in LibraryTableColumns.tsx as well.

File: frontend/src/components/samples/details/SampleDetailsProcessMeasurements.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (UlysseFG) [reply frontend/src/components/samples/details/SampleDetailsProcessMeasurements.js:51]: Not sure what can set a variable to null ... but yeah possibly ... I don't think we'll knowingly set it to null but it is another real false.

[the remark this answers is not in the document]
Comment (ckostiw) [reply frontend/src/components/samples/details/SampleDetailsProcessMeasurements.js:51]: Actually, it will be null if no value has been set, 0 if the volume is zero, or undefined if the sample is still being fetched and it completely missing the volume property. Maybe we should write a `renderFloat()` function to take care of all that complexity.

**Review by ckostiw — CHANGES_REQUESTED** [review]
LibraryTableColumns.tsx should be fixed too

**Comment by UlysseFG** [comment]
I would make it renderNumber since any numeric value will face the same issue.

File: frontend/src/components/samples/details/SampleDetailsProcessMeasurements.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (nafiz1001) [reply frontend/src/components/samples/details/SampleDetailsProcessMeasurements.js:51]: Nah I believe we should add more bloat by installing another package to take care of that 😉

[the remark this answers is not in the document]
Comment (nafiz1001) [reply frontend/src/components/samples/details/SampleDetailsProcessMeasurements.js:51]: Also what should `renderFloat()` show if they're `undefined` or `null`.

**Review by ckostiw — APPROVED** [review]
